### PR TITLE
New design for subscription page

### PR DESCRIPTION
**Pull request by Alevsk**
## How it looks

Screen Shot 2021-01-28 at 00 17 22
Screen Shot 2021-01-28 at 00 17 29
Screen Shot 2021-01-28 at 00 17 36

## Mockups

- https://xd.adobe.com/view/deb40630-9733-4568-8787-6b82d43ccbf7-8be7/
- https://xd.adobe.com/view/d0669d9d-f8c8-42b7-a1ac-e586988568a4-8535/screen/d9aacc1a-f476-4a50-9f5b-216a4cb14fb9
- https://xd.adobe.com/view/a7f71179-4113-4e6b-9cd0-969dfdad790e-f2a4/screen/20e8c37e-515e-47b6-9b1f-72b3e97d6184

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/License/ActivationModal.tsx [hunk 1]
```diff
@@ -37,16 +38,32 @@ const styles = (theme: Theme) =>
...
     },
     subnetLoginForm: {
       padding: "10px 0px 10px 10px",
+      opacity: 0.5,
+      "&:hover": { opacity: 1 },
     },
-    licenseKeyField: {
-      marginBottom: 20,
-    },
+    licenseKeyField: {},
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/ActivationModal.tsx:49]: If it's empty we can remove

**Review by bexsoft — CHANGES_REQUESTED** [review]
Moe comments bellow:

**Comment by bexsoft** [comment]
Screen Shot 2021-01-28 at 13 01 23
Can we remove the opacity effect?, also the activate button should be greyed out when one field is empty to have consistency

**Comment by bexsoft** [comment]
Screen Shot 2021-01-28 at 13 03 07
Can we remove the borders & styles for activate button?

**Comment by bexsoft** [comment]
Can we please make `portal-ui/public/agpl.png` to be a svg file?

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**